### PR TITLE
Schedule doc conflict cleanup every week

### PR DESCRIPTION
**Pull request by TylerSheffels**
@snopoke Should we have a better way of resolving doc conflicts than just removing the conflicting document?

http://manage.dimagi.com/default.asp?182427 cc: @emord

### CODE REVIEW

File: corehq/apps/cleanup/management/commands/delete_doc_conflicts.py [hunk 1]
```diff
@@ -9,18 +10,18 @@ def bulk_delete(db, docs):
     if not docs:
         return
 
-    print("Deleting {} doc revisions".format(len(docs)))
+    logging.info("Deleting {} doc revisions".format(len(docs)))
```
Comment (TylerSheffels) [inline corehq/apps/cleanup/management/commands/delete_doc_conflicts.py]: Do I need to pick logging handler e.g. logging.getLogger() or is the main one fine here? Is there a general policy?

Comment (snopoke) [inline corehq/apps/cleanup/management/commands/delete_doc_conflicts.py]: I think its fine in management command but not if the code is run in a service

**Comment by snopoke** [comment]
Seems OK, but probably best to change the logger

**Comment by TylerSheffels** [comment]
@snopoke updated

**Pull request merged**